addons/mail/static/tests/thread/unread_messages_banner.test.js

```diff
@@ -68,49 +68,6 @@ test("mark thread as read from unread messages banner", async () => {
     await contains(".o-mail-Thread-jumpToUnread", { count: 0 });
 });
 
-test("scroll to the first unread message (slow ref registration)", async () => {
```

Comment:
This is this test moved/removed?

It looks like it is another flow, we should probably duplicate it instead.

Reply:
It's the same flow: the message highlight occurs when the reference registration is delayed. However, this test was known to be non-deterministic, and using the unread message banner to test the highlight case is more complicated than it should be. I took the opportunity to make this test more generic and to focus on testing what was actually intended: the highlight itself.